zbus_xmlgen/src/lib.rs

```diff
@@ -204,7 +204,7 @@ impl<'i> GenTrait<'i> {
         let mut props = iface.properties().to_vec();
         props.sort_by(|a, b| a.name().partial_cmp(&b.name()).unwrap());
         for p in props {
-            let name = to_identifier(&to_snakecase(p.name().as_str()));
+            let name = to_identifier(&to_snakecase(p.name().as_str())) + "_property_";
```

Comment:
while adding `signal` and `property` prefix resolve the issues in question, doing so unconditionally even when there's no conflict, isn't nice. The would be a pretty big behavior change and gives no benefit to cases where there are no conflicts. 

I think I good solution here would only change generation of conflicted parts and leave the rest, as is.

Reply:
Thanks for the feedback. I think conditionally adding the suffix in case of conflict would work.

However, I also like the new convention as it gives a clear distinction between the three types.
Would you accept a command line option that always enforces the more verbose convention even when no conflict is detected?

Reply:
> However, I also like the new convention as it gives a clear distinction between the three types.

Sorry, no. The attributes already make it clear, what's a property, a signal and the lack of any attribute means it's a method. There is no need for further clarity.

> Would you accept a command line option that always enforces the more verbose convention even when no conflict is detected?

If you can provide compelling reason for it, we can discuss it. :)

Reply:
I update the PR to just add `_` until there is no more collision.